src/main/resources/templates/datewarehouse/etl-jobs-config.json

```diff
@@ -187,17 +505,37 @@
 		"sourcePath" : "log.sagebase.org/userGroupSnapshots/",
 		"tableDescriptor": {
 			"name": "usergroupsnapshots",
-			"columns": {
-				"change_timestamp": "timestamp",
-				"snapshot_timestamp": "timestamp",
-				"id": "bigint",
-				"is_individual": "boolean",
-				"created_on": "timestamp"
-
+			"description" : "This table lists all principals (individual users and groups of users). (A group is the low-level object underlying a team, much like a file handle is the low-level object underlying a file entity.) In addition to explicit users and teams, principals in Synapse include the anonymous user, the implicit group of all authenticated users, and the implicit public group which includes all users, authenticated or not. Note: Snapshots are also taken periodically and independently of the changes. The snapshot_timestamp records when the snapshot was taken.",
```

Comment:
Same question as above for distinguishing the difference between snapshots created via node creation/update/deletion vs the periodic snapshot taking.

Why does this snapshot exist when there are individual team, team member and user profile snapshots?

Reply:
usergroup  are created to joined ACLs . Permission can be granted to a team or an individual, the join is made on usergroup, and from there we join to principal or team depending on is_individual. Thats why we have usergroup and we take their snapshots.